frontend/core-ui/src/modules/automations/utils/automationBuilderUtils.ts

```
@@ -402,3 +401,150 @@ export const generateSendEmailRecipientMails = ({
+    };
+  }
+
+  const position = reactFlowInstance.screenToFlowPosition({
+    x: event.clientX,
+    y: event.clientY,
+  });
+
+  const fieldName = `detail.${nodeType}s` as
+    | 'detail.triggers'
+    | 'detail.actions';
+  let existingNodes = nodeType === 'trigger' ? triggers : actions;
```

ESLint

[error] 'existingNodes' is never reassigned. Use 'const' instead.

(prefer-const)